### PR TITLE
LPS-96579 getCollection companyId parameter fixup

### PR DESCRIPTION
**Pull request by zoltancsaszi**

### CODE REVIEW

**Comment by zoltancsaszi** [comment]
ci:test:sf

**Comment by zoltancsaszi** [comment]
ci:test:change-tracking

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/ProcessResourceImpl.java [hunk 1]
```diff
@@ -67,6 +80,20 @@ public Process getProcess(Long processId) throws Exception {
...
+			Long companyId, String keywords, ProcessType type, Long userId,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		List<Process> processes = _getProcesses(
+			_getCTProcesses(
+				companyId, GetterUtil.getLong(userId), keywords, type,
+				pagination, sorts));
+
+		return Page.of(processes, pagination, processes.size());
```
Comment (nhpatt) [inline modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/ProcessResourceImpl.java]: If you don't need to return the requested pagination (the last parameter of Page.of is the totalCount), there is a method lile Page.of(processes)

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/ProcessResourceImpl.java [hunk 2]
```diff
@@ -79,6 +106,48 @@ public Process getProcess(Long processId) throws Exception {
...
+				_ctEngineManager.getLatestCTProcessOptional(companyId);
+
+			ctProcesses = latestCTProcessOptional.map(
+				Collections::singletonList
+			).orElse(
+				Collections.emptyList()
+			);
+		}
+		else {
+			int status = _getStatus(type);
```
Comment (nhpatt) [inline modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/ProcessResourceImpl.java]: has to be inlined

File: modules/apps/change-tracking/change-tracking-rest-client/src/main/java/com/liferay/change/tracking/rest/client/resource/v1_0/CollectionResource.java [hunk 3]
```diff
@@ -289,11 +289,15 @@ public void deleteCollection(Long companyId, Long collectionId)
 
 			httpInvoker.httpMethod(HttpInvoker.HttpMethod.DELETE);
 
+			if (companyId != null) {
```
Comment (nhpatt) [inline modules/apps/change-tracking/change-tracking-rest-client/src/main/java/com/liferay/change/tracking/rest/client/resource/v1_0/CollectionResource.java]: Generator changes has to be in another commit

**Comment by nhpatt** [comment]
Thanks! :)
